scripts/repin.sh

```diff
@@ -10,7 +10,7 @@ set -e
 eval "$(conda shell.bash hook)"
 
 # Create openwpm env with unpinned yaml file
```

Comment:
We should add a note about the PYTHONNOUSERSITE=True addition (can just point to the discussion on this PR) so that future devs can know why it's here.

Reply:
Will do!